.github/workflows/test.yml

```diff
@@ -15,34 +15,34 @@ jobs:
     strategy:
       matrix:
         node-version: [12.x, 14.x, 16.x]
-    
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v2
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Install dependencies
-      run: npm ci
-    - name: Test
-      run: npm test
-    - name: Test examples
-      run: |
-        cd examples/build-and-serve
-        npm i
-        npm run build:ci
-    - name: Test in the oldest Google Closure Tools
-      run: |
-        npm i --no-save google-closure-compiler@20180910 google-closure-deps@20190325
-        OLDEST_COMPILER=1 npm run unit
-    - name: Test examples in the oldest Google Closure Tools
-      run: |
-        cd examples/build-and-serve
-        npm i --no-save google-closure-library@20180910
-        npm run build:ci 
-    - name: Report coverage to codecov
-      if: matrix.node-version == '12.x'
-      env:
-        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-      run: npx codecov
 
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
```

Comment:
The only real change is in this line. Other lines were changed by Prettier via my text editor formatter.